src/usb/uf2/uf2cfg.h

```diff
@@ -1,9 +1,15 @@
 #include "boards.h"
 #include "dfu_types.h"
 
-// Family ID for updating Application
+// Legacy Family ID for updating Application
 #define CFG_UF2_FAMILY_APP_ID     0xADA52840
 
+// Family ID for board-specific Application
+#if defined(USB_DESC_VID) && defined(USB_DESC_UF2_PID) && USB_DESC_VID && USB_DESC_UF2_PID
```

Comment:
let's make the CFG_UF2_BOARD_APP_ID always defined. This repo doesn't accept nrf52840/833 that doesn't have VID/PID.

Reply:
Ok, fixed with commit aa8cd83.

Reply:
you could keep ifndef here to make CI happy for now. Please remove other in ghostfat.c, that file is not compiled with 832